certified-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -1,324 +1,932 @@
 {
-  "swagger": "2.0",
-  "info": {
-    "title": "GroupMgr",
-    "description": "A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
-    "version": "1.0",
-	"contact": {
-      "name": "GroupMgr Helpdesk",
-      "url": "https://www.vanroey.be/support/",
-      "email": "support@groupmgr.com"
-    }
-  },
-  "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
-  "basePath": "/",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [],
-  "produces": [],
-  "paths": {
-    "/api/webhookrequest": {
-      "x-ms-notification-content": {
-        "description": "Caught group creation",
-        "schema": {
-          "type": "object",
-          "properties": {
-            "DisplayName": {
-              "type": "string",
-              "description": "The display name of the group.",
-              "title": "Display name"
+    "swagger": "2.0",
+    "info": {
+        "title": "GroupMgr",
+        "description": "A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
+        "version": "2.3",
+        "contact": {
+            "name": "GroupMgr Helpdesk",
+            "url": "https://www.vanroey.be/en/support/",
+            "email": "support@groupmgr.com"
+        }
+    },
+    "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "x-ms-connector-metadata": [{
+            "propertyName": "Website",
+            "propertyValue": "https://groupmgr.com/"
+        }, {
+            "propertyName": "Privacy policy",
+            "propertyValue": "https://groupmgr.com/privacy"
+        }, {
+            "propertyName": "Categories",
+            "propertyValue": "IT Operations;Productivity"
+        }
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/webhookrequest/GroupRequested": {
+            "x-ms-notification-content": {
+                "description": "Caught group request.",
+                "schema": {
+                    "$ref": "#/definitions/GroupCompact"
+                }
             },
-            "GroupType": {
-              "type": "string",
-              "description": "The group type.",
-              "title": "Type"
+            "post": {
+                "responses": {
+                    "default": {
```

Comment:
@Jwaegebaert We recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.